python/repositories.bzl

```diff
@@ -179,6 +179,16 @@ def _python_repository_impl(rctx):
             "share/**",
         ]
 
+    python_wrapper = "python_wrapper.sh"
+    python_wrapper_content = """\
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+
+external/{repo_name}/{python_bin} -B -s -I "$@"
```

Comment:
Do we want -S (upper case) as well?

Reply:
I don't think this would harm in the case a custom toolchain from this repo is used. In the case a system python is used, this may even be desirable.